### ISSUE TITLE
Retests for Windoze (assigned to Brian)

### ISSUE THREAD

**Issue by emyoulation** [cdhorn/LinkedView#118]
There are a couple idiosyncracies on Windoze & not Linux.

Once I can download again, I'll re-test with the new code, and if they still exist, look through the other known Windoze issue. If that doesn't help, we can see if Paul can make a suggestion.

But it needs a place to track progress.

So my testing targets are:
* [Toolbar labels are non-standard #113](https://github.com/cdhorn/LinkedView/issues/113)  <br />_fix works in some categories, not others_

*  [Minimize of Copy Page affect application, not just spawned window #116](https://github.com/cdhorn/LinkedView/issues/116) <br />_works in Linux but not Windoze. macOS?_

**Comment by emyoulation** [comment]
Only 1 more day more day of jail for breaking the data limit. Then I'll be on parole for 5 gigs. But I'm a repeat offender. It's habitual so it is only a matter of time.

**Comment by cdhorn** [comment]
I forget but I think I need a plugin to check #113 so didn't glance at it tonight. 

With regards to #116 I think this is a side effect of how GTK interacts with the Windows window manager. I would be curious how this behaves on MacOS. This is outside my sphere of knowledge to fix, if there even is a fix. Perhaps Paul knows.

With regards to #119 I don't seem to be seeing this behaviour. Did this go away for you with 5.1.5?

**Comment by cdhorn** [comment]
Yes #113 I think I fixed in some other commit some time back. Let me know if you see something amiss there.
